.github/workflows/deploy_website.yaml

```
@@ -114,3 +119,8 @@ jobs:
           strapi_endpoint: ${{ vars.STRAPI_ENDPOINT }}
           strapi_api_token: ${{ secrets.STRAPI_API_TOKEN }}
           fetch_from_strapi: ${{ vars.FETCH_FROM_STRAPI }}
+
+      - name: Chatbot Vector Index
+        uses: ./.github/actions/chatbot
```

This makes the entire step to run on the self-hosted runner in AWS. Also, this takes into account the target environment.
```suggestion
        uses: ./.github/actions/chatbot
        runs-on: codebuild-${{ inputs.environment }}-github-runner-${{ github.run_id }}-${{ github.run_attempt }}
```